tests/unittests/label_tests.py

```
@@ -859,6 +862,154 @@ def test_perform_nms(self):
         self.assertListEqual(ids3, [id2])
 
 
+class ExportMaskTests(unittest.TestCase):
+    """Tests for Detection.export_mask and Segmentation.export_mask."""
+
+    _MASK = np.ones((5, 5), dtype=np.uint8) * 255
+    _ZEROS = np.zeros((5, 5), dtype=np.uint8)
+
+    def _make_detection(self, mask=None, mask_path=None):
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `_make_detection`

(ANN202)